packages/neuron-ui/src/components/WalletWizard/index.tsx

```diff
@@ -259,12 +260,13 @@ const Mnemonic = ({ state = initState, rootPath = '/wizard/', dispatch, isSettin
 
   const onBack = useCallback(() => {
     changeStep(v => v - 1)
+    const isSettings = searchParams.get('isSettings') === '1'
```

Comment:
I mean we can remove `isSettings` variable. Because `settings` page is also in the main window.

Reply:
<img width="531" alt="image" src="https://user-images.githubusercontent.com/11756869/234519087-cba7e8cf-f372-4df5-a6b8-25b30a24e4df.png">

when 【create new wallet】from settings page, we need go back to settings. so we need isSettings flag